docs/install/tutorials/rpi.rst

```diff
@@ -269,9 +269,14 @@ Installing Kolibri
       sudo su -c 'echo "deb http://ppa.launchpad.net/learningequality/kolibri/ubuntu xenial main" > /etc/apt/sources.list.d/learningequality-ubuntu-kolibri-xenial.list'
       sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys DC5BAA93F9E4AE4F0411F97C74F88ADB3194DD81
       sudo apt update
-      sudo apt install kolibri
 
-   When asked questions during the installation, it is recommended that you use the default ``pi`` user for running Kolibri because it will have access to USB devices.
+#. Install ``kolibri`` and ``kolibri-server``:
+
+   .. code-block:: bash
+
+      sudo apt install kolibri kolibri-server
+
+   When asked questions during the installation, use the default TCP port ``8080`` for Kolibri (port 80 will be activated :ref:`later in this tutorial <nginx_custom_domain>`). The default ``pi`` user for running Kolibri because it will have access to USB devices.
```

Comment:
When prompted, use the default TCP port `8080` for Kolibri because port `80` will be activated later in this tutorial. Use the default ``pi`` user  so that it will have access to USB devices.

Reply:
Agreed, have rephrased in f755a99